spdx/__init__.py

```diff
@@ -1,13 +1 @@
-# Copyright 2014 Ahmed H. Ismail
-
-#    Licensed under the Apache License, Version 2.0 (the "License");
-#    you may not use this file except in compliance with the License.
-#    You may obtain a copy of the License at
-
-#        http://www.apache.org/licenses/LICENSE-2.0
-
-#    Unless required by applicable law or agreed to in writing, software
-#    distributed under the License is distributed on an "AS IS" BASIS,
-#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#    See the License for the specific language governing permissions and
-#    limitations under the License.
\ No newline at end of file
+__import__('pkg_resources').declare_namespace(__name__)
```

Comment:
Note that this makes spdx a "namespace" .... this is provisional and to eventually avoid conflicts with other packages published on Pypi that already use the spdx namespace.